<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>2</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>650</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1100</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Customer Management System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/window.png</normaloff>:/images/window.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="windowIcon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/HOME.png</normaloff>:/images/HOME.png</iconset>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Home"/>
   <addaction name="separator"/>
   <addaction name="action_Customer"/>
   <addaction name="separator"/>
   <addaction name="action_Product"/>
   <addaction name="separator"/>
   <addaction name="action_Order"/>
   <addaction name="separator"/>
   <addaction name="action_Chat"/>
   <addaction name="separator"/>
   <addaction name="action_Quit"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_Customer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/CUSTOMER.png</normaloff>:/images/CUSTOMER.png</iconset>
   </property>
   <property name="text">
    <string>&amp;CUSTOMER</string>
   </property>
  </action>
  <action name="action_Product">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/PRODUCT.png</normaloff>:/images/PRODUCT.png</iconset>
   </property>
   <property name="text">
    <string>&amp;PRODUCT</string>
   </property>
  </action>
  <action name="action_Order">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/ORDER.png</normaloff>:/images/ORDER.png</iconset>
   </property>
   <property name="text">
    <string>&amp;ORDER</string>
   </property>
  </action>
  <action name="action_Home">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/HOME.png</normaloff>:/images/HOME.png</iconset>
   </property>
   <property name="text">
    <string>&amp;HOME</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/QUIT.png</normaloff>:/images/QUIT.png</iconset>
   </property>
   <property name="text">
    <string>&amp;QUIT</string>
   </property>
  </action>
  <action name="action_Chat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/CHAT.png</normaloff>:/images/CHAT.png</iconset>
   </property>
   <property name="text">
    <string>&amp;CHAT</string>
   </property>
   <property name="toolTip">
    <string>CHAT</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
